.github/workflows/scan.yml

```diff
@@ -20,3 +20,31 @@ jobs:
         go-version: 1.19
     - name: Run verify container script
       run: make verify-container-images
+  scan-release-1-3:
+    name: Trivy (release-1.3)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag=v3.1.0
```

Comment:
Can you please change all checkout actions to

uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # tag=v3.3.0

(also the main one. There was a bump PR but it didn't bump the tag corretly because the previous hash didn't match the documented v3.1.0 tag: https://github.com/kubernetes-sigs/cluster-api/pull/7881/files#diff-246cd0c2f7db532638dd80a92ac011f49b3d26038983a4c0169ea8f8a5c39280)